src/Modules/Payments/Application/PriceListItems/CreatePriceListItem/CreatePriceListItemCommand.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using CompanyName.MyMeetings.Modules.Payments.Application.Configuration.Commands;
+using Newtonsoft.Json;
+
+namespace CompanyName.MyMeetings.Modules.Payments.Application.PriceListItems.CreatePriceListItem
+{
+    public class CreatePriceListItemCommand : InternalCommandBase<Guid>
```

Comment:
This is not Internal Command. It is normal Command (invoked by User through API) so it should inherit from `CommandBase<Guid>`

Reply:
Fixed